mgmtd/mgmt_be_adapter.c

```diff
@@ -65,6 +65,23 @@ struct mgmt_be_xpath_map {
  * above map as well.
  */
 
+static const char *const zebra_config_xpaths[] = {
+	"/frr-affinity-map:lib",
+	"/frr-filter:lib",
+	"/frr-route-map:lib",
+	"/frr-zebra:zebra",
+	"/frr-interface:lib",
+	"/frr-vrf:lib",
+	NULL,
+};
+
+static const char *const zebra_oper_xpaths[] = {
+	"/frr-interface:lib/interface",
```

Comment:
Can we add `/frr-zebra:zebra` to the interface oper xpath?

Reply:
Sure.